### DISCUSSION TITLE
Use of "Open Source"

### DISCUSSION THREAD

**Discussion by ssddanbrown** [investbrainapp/investbrain#40] [General]
Hello :wave:,

Just come across this project after watching "This Week in Self-Hosted".

Saw it was marketed/advertised as "open source", but this is using a license that may not be widely considered open source due to limitations of use via the license (preventing commercial use) which won't meet [the OSD](https://opensource.org/osd).
Just thought I'd raise since otherwise this could be potentially misleading to many, who also may assume this provides open use, modification & distribution as an open source license typically would.

Generally, this kind of licensing may be regarded as "source available", although there are other options and sub-categories in this area.

If you want for further information/context on this, I have a couple of blogposts [here](https://danb.me/blog/why-open-source-term-is-important/) and [here](https://danb.me/blog/open-source-available-distinction/).

As an extra note, creative commons don't recommend using their licenses for software [for the reasons provided here](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software).

**Comment by hackeresq** [comment]
Appreciate the note @ssddanbrown. And great write ups on your blog! 

I'll definitely have to review the options. In the meanwhile, leaving this open for continued discussion.
